### PR TITLE
Update list.txt with latest spammers (11/11/2022)

### PR DESCRIPTION
**Pull request by JasonD94**
This PR updates list.txt with the latest spammers from my domain level block list as of 11/11/2022

### CODE REVIEW

File: list.txt [hunk 1]
```diff
@@ -1,45 +1,65 @@
...
+adventglobal.com
 aerotek.com
+agamasolutions.com
 agilepeoplestaffing.com
 agreeya.com
 aitrg.com
 alissoftware.com
 altaits.com
+alz.org
+amazon.com
```
Comment (JasonD94) [inline list.txt]: Ah I should probably remove this one, though to be fair Amazon recruiters are insane.

Comment (johnceloria) [reply list.txt:19]: You understand that this will filter all emails coming from @amazon.com, right?

Comment (jceloria) [reply list.txt]: You understand that this will filter all emails coming from @amazon.com, right?

Comment (JasonD94) [reply list.txt]: Yeah, I just removed that one.

**Pull request closed without merging**